Source/Scene/ExpressionType.js

```diff
@@ -0,0 +1,19 @@
+/*global define*/
+define([
+    '../Core/freezeObject'
+], function(
+    freezeObject) {
+    'use strict';
+
+    /**
+     * @private
+     */
+    var ExpressionType = {
```

Comment:
We need to be rigorous with terminology.  This is not an expression type.  This is the type of a node in an AST.